<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>InfoWidget</class>
 <widget class="QWidget" name="InfoWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>81</width>
    <height>306</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="shortLabel">
     <property name="text">
      <string>shortLabel</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="syspathLabel">
     <property name="text">
      <string>syspathLabel</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="subsysLabel">
     <property name="text">
      <string>subsysLabel</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="locationLabel">
     <property name="text">
      <string>locationLabel</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="driverLabel">
     <property name="text">
      <string>driverLabel</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="friendlyLabel">
     <property name="text">
      <string>friendlyLabel</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="descrLabel">
     <property name="text">
      <string>descrLabel</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="mfgLabel">
     <property name="text">
      <string>mfgLabel</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="vidLabel">
     <property name="text">
      <string>vidLabel</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="pidLabel">
     <property name="text">
      <string>pidLabel</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="serviceLabel">
     <property name="text">
      <string>serviceLabel</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="busLabel">
     <property name="text">
      <string>busLabel</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="revLabel">
     <property name="text">
      <string>revLabel</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="exLabel">
     <property name="text">
      <string>exLabel</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="busyLabel">
     <property name="text">
      <string>busyLabel</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
